<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estação Metereológica</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>322</width>
      <height>189</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="gridVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>321</width>
      <height>130</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QTimeEdit" name="HoraFinal">
       <property name="locale">
        <locale language="Portuguese" country="Brazil"/>
       </property>
       <property name="maximumDateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::HourSection</enum>
       </property>
       <property name="displayFormat">
        <string notr="true">hh:mm</string>
       </property>
       <property name="currentSectionIndex">
        <number>0</number>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
       <property name="time">
        <time>
         <hour>14</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="BuscarButton">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="LimparButton">
       <property name="text">
        <string>Limpar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTimeEdit" name="HoraInicial">
       <property name="locale">
        <locale language="Portuguese" country="Brazil"/>
       </property>
       <property name="maximumTime">
        <time>
         <hour>12</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
       <property name="displayFormat">
        <string notr="true">hh:mm</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>71</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelMaiorUmidade">
       <property name="text">
        <string>Maior:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMenorUmidade">
       <property name="text">
        <string>Menor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMediaUmidade">
       <property name="text">
        <string>Média:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUltimoUmidade">
       <property name="text">
        <string>Último:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>71</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="maiorumidade">
       <property name="text">
        <string>80%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="menorumidade">
       <property name="text">
        <string>75%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mediaumidade">
       <property name="text">
        <string>67,5%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ultimaumidade">
       <property name="text">
        <string>79%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="Umidade_label">
       <property name="text">
        <string>Umidade</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>71</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="maior_label">
       <property name="text">
        <string>Maior:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="menor_label">
       <property name="text">
        <string>Menor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="media_label">
       <property name="text">
        <string>Média:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ultimo_label">
       <property name="text">
        <string>Último:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>71</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="maiortemperatura">
       <property name="text">
        <string>24ºC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="menortemperatura">
       <property name="text">
        <string>24ºC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mediatemperatura">
       <property name="text">
        <string>24ºC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ultimatemperatura">
       <property name="text">
        <string>24ºC</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="Temperatura_label">
       <property name="text">
        <string>Temperatura</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_whatisshowing">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>8</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>gridLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>verticalLayoutWidget_4</zorder>
   <zorder>verticalLayoutWidget_5</zorder>
   <zorder>verticalLayoutWidget_6</zorder>
   <zorder>label_whatisshowing</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
